src/apps/moola/debtTokens.ts

```diff
@@ -0,0 +1,76 @@
+import { Address } from 'viem'
+
+export interface DebtTokenDefinition {
+  baseTokenAddress: Address
+  debtTokenAddress: Address
+  imageUrl: string
+  title: string
+  description: string
+}
+
+export const MOOLA_DEBT_TOKENS: DebtTokenDefinition[] = [
+  {
+    baseTokenAddress: '0xd8763cba276a3738e6de85b4b3bf5fded6d6ca73',
+    debtTokenAddress: '0xfb6c830c13d8322b31b282ef1fe85cbb669d9ae8',
+    imageUrl:
+      'https://raw.githubusercontent.com/valora-inc/address-metadata/main/assets/tokens/mcEUR.png',
```

Comment:
We can use the Moola logo for all of them for now.
```suggestion
    imageUrl:
      'https://raw.githubusercontent.com/valora-inc/dapp-list/main/assets/moola.png',
```